scipy/stats/kde.py

```diff
@@ -209,14 +211,14 @@ def evaluate(self, points):
             # there are more points than data, so loop over data
             for i in range(self.n):
                 diff = self.dataset[:, i, newaxis] - points
-                tdiff = dot(self.inv_cov, diff)
+                tdiff = linalg.cho_solve(self.covariance_chol, diff)
```

Comment:
In the mentioned test, we fail here because of the reliance of this method on `self.inv_cov`.